app-config.yaml

```diff
@@ -52,16 +52,23 @@ proxy:
 # Note: After experimenting with basic setup, use CI/CD to generate docs
 # and an external cloud storage when deploying TechDocs for production use-case.
 # https://backstage.io/docs/features/techdocs/how-to-guides#how-to-migrate-from-techdocs-basic-to-recommended-deployment-approach
+# techdocs:
+#   builder: 'external'
+#   publisher:
+#     type: 'awsS3'
+#     awsS3:
+#       bucketName: ${AWS_BUCKET_NAME}
+#       region: ${AWS_BUCKET_REGION}
+#       credentials:
+#         accessKeyId: ${AWS_ACCESS_KEY_ID}
+#         secretAccessKey: ${AWS_ACCESS_KEY_SECRET}
+
 techdocs:
```

Comment:
Is the intention here to have the code be local ready rather than having changes necessary when you pull? Does this same app-config work as expected in dev?

Reply:
This config in the repo is used during local deploys only. The K8s deployment uses a template file found in our config repo to generate an app config file during deployment using a Helm values file.

That being said, when we switch to envconsul, we will no longer need the config template for deploys and need to figure out how we use this config both locally and in our envs. We can all sync up on this soon.

Just some initial thoughts, kicking around the idea of using a tool to mock s3 service locally or use an app-config-local.yaml in the local startup script. Sure there are other options we can discuss as well.

Reply:
Localstack is one option to mock a variety of AWS services